presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaSecurityConfig.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import io.airlift.configuration.LegacyConfig;
+import io.airlift.configuration.validation.FileExists;
+import org.apache.kafka.common.config.SslConfigs;
+
+import java.util.Locale;
+import java.util.Properties;
+
+/**
+ *  {@KafkaSecurityConfig} manages Kafka configuration related to connection security. I.e. if the security protocol
+ *  'SSL' is being used. If so, additional properties might be supplied, i.e. location and password of the truststore
+ *  and keystore to be used.
+ */
+public class KafkaSecurityConfig
+{
+    private Properties props;
+
+    private String sslKeystoreFile;
+    private String sslKeystorePassword;
+    private String sslKeyPassword;
+    private String sslTruststoreFile;
+    private String sslTruststorePassword;
+    private KafkaEndpointIdentificationAlgorithm sslEndpointIdentificationAlgorithm;
+
+    @Config("kafka.keystore-file")
```

Comment:
Why this one is better than the one below?

Reply:
Don't get the question?

Reply:
Why we are going to intruce `@Config` along with `@LegacyConfig`? I understand the the one in `@LegacyConfig` is used else where, so why to rename it and not keep that one?

Reply:
`@LegacyConfig` with the property names of Starburst Presto was explicitly requested by another reviewer on this PR

Reply:
Why not to use the ones that comes from Starburst?

Reply:
since they don't fit the prestosql convention for property names... (also the `kafka-clients` property names does not match the convention) please see comments from `losipiuk` 
(`@LegacyConfig` should be added so that having them set with Starburst-names works as well)

Reply:
Got it thanks!